### PR TITLE
CUDA documentation tweaks

### PR DESCRIPTION
**Pull request by elistevens**
Addresses comments on previous PR.

### CODE REVIEW

File: docs/source/notes/serialization.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    torch.save(the_model, PATH)
+
+Then later::
+
+    the_model = torch.load(PATH))
+
+The second relies on both the shape of the model, as well as the class
+definition. This results in it being more fragile, since if the source code
+of the class changes, the model will no longer load.
```
Comment (apaszke) [inline docs/source/notes/serialization.rst]: That's not true. A model with some changes in the source can still load and work.

**Comment by apaszke** [comment]
I added the fixes to your branch. Let's wait for the lint check to make sure it's all ok, and we can merge this. Thanks!

**Comment by elistevens** [comment]
Ah, okay. It was described to me as more fragile with more points of
failure than saving the parameters. Can you please describe what that
fragility entails?

**Comment by elistevens** [comment]
Ah, I see. Thanks!

**Pull request merged**